src/gui/viewer.cpp

```diff
@@ -573,6 +669,23 @@ void InfoWidget::setup_ui() {
 ImageViewWidget::ImageViewWidget(Document*, QWidget* pParent)
 	: QWidget(pParent) {
 	setMinimumSize(256, 256);
+
+
+	// Make the checkerboard pattern default image.
+	// This is done here rather than in paint for performance's sake, as the checkerboard will never change.
+	checkerboard = QImage(checkerboard_size, checkerboard_size, QImage::Format_RGB32);
+	QRgb val;
+	val = qRgb(55, 60, 65);
+	QRgb val2;
+	val2 = qRgb(60, 64, 68);
```

Comment:
nit:

```suggestion
	QRgb val = qRgb(55, 60, 65);
	QRgb val2 = qRgb(60, 64, 68);
```

Personally I'd like these names to be better as well, but it's fine